store/dhash/dhash.go

```diff
@@ -64,7 +67,7 @@ func EncryptAES(payload, passphrase []byte) ([]byte, []byte, error) {
 	// Create initialization vector (nonse) to be used during encryption
```

Comment:
`nonse` ?

Reply:
We derive nonse from the passphrase so that we can deterministically produce the same result of `hash(mh) -> EncPeerID` given the same multihash / peerID.

Reply:
I was just pointing out the typo.